stardis/io/model/marcs.py

```diff
@@ -48,7 +49,14 @@ def to_geometry(self):
             reference_r = self.metadata["radius"]
         return Radial1DGeometry(r, reference_r)
 
-    def to_composition(self, atom_data, final_atomic_number):
+    def to_composition(
+        self,
+        atom_data,
+        final_atomic_number,
+        composition_source,
+        helium_mass_frac_Y,
```

Comment:
I'm not sure about having optional args that only function when another arg is set. I would probably rather the composition_source be a dict?